<?xml version="1.0" encoding="UTF-8"?>
<bajadoc version="2.0" createdBy="niagara-baja-doclet-1.0.9" createdAt="03-May-2024" createdOn="883e7f7a9875">
<class module="seriesTransform" runtimeProfile="rt" qualifiedName="javax.baja.seriestransform.exceptions.SchemaException" name="SchemaException" packageName="javax.baja.seriestransform.exceptions" public="true" category="exception">
<description>
The Schema exception indicates an exception involving the structure or &#xa; resolving of a schema for a transform graph or graph node.
</description>
<tag name="@author">J. Spangler</tag>
<tag name="@creation">Apr 30, 2012</tag>
<extends>
<type class="javax.baja.seriestransform.exceptions.TransformException"/>
</extends>
<!-- javax.baja.seriestransform.exceptions.SchemaException(java.lang.String, java.lang.Throwable) -->
<constructor name="SchemaException" public="true">
<parameter name="msg">
<type class="java.lang.String"/>
</parameter>
<parameter name="cause">
<type class="java.lang.Throwable"/>
</parameter>
<description>
Constructor with specified message and nested exception.
</description>
</constructor>

<!-- javax.baja.seriestransform.exceptions.SchemaException(java.lang.Throwable) -->
<constructor name="SchemaException" public="true">
<parameter name="cause">
<type class="java.lang.Throwable"/>
</parameter>
<description>
Constructor with specified nested exception.
</description>
</constructor>

<!-- javax.baja.seriestransform.exceptions.SchemaException(java.lang.String) -->
<constructor name="SchemaException" public="true">
<parameter name="msg">
<type class="java.lang.String"/>
</parameter>
<description>
Constructor with specified message.
</description>
</constructor>

<!-- javax.baja.seriestransform.exceptions.SchemaException() -->
<constructor name="SchemaException" public="true">
<description>
No argument constructor.
</description>
</constructor>

</class>
</bajadoc>
